codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketClientHandshaker.java

```diff
@@ -140,11 +140,14 @@ private void setActualSubprotocol(String actualSubprotocol) {
     }
 
     /**
+     * @deprecated Use {@link #handshake(ChannelHandlerContext)}.
```

Comment:
IIRC, the `@deprecated` tag has to come after the summary text:

```java
/**
 * Begins the opening handshake.
 *
 * @deprecated Use .... instead.
 */
```

Otherwise, the summary text becomes the part of the deprecation warning text.